Options.py

```diff
@@ -1412,6 +1418,115 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link["item_pool"] = list(pool)
 
 
+class PlandoItem(typing.NamedTuple):
+    items: typing.Union[typing.List[str], typing.Dict[str, typing.Any]]
+    locations: typing.List[str]
+    world: typing.Union[int, str, bool, None, typing.Iterable[str], typing.Set[int]] = False
+    from_pool: bool = True
+    force: typing.Union[bool, typing.Literal["silent"]] = "silent"
+    count: typing.Union[int, bool, typing.Dict[str, int]] = False
```

Comment:
```suggestion
    items: list[str] | dict[str, typing.Any]
    locations: list[str]
    world: int | str | bool | None | typing.Iterable[str] | set[int] = False
    from_pool: bool = True
    force: bool | typing.Literal["silent"] = "silent"
    count: int | bool | dict[str, int] = False
```
This can be a lot cleaner now with 3.8 and 3.9 gone.